### ISSUE TITLE
Ability to open currently viewed thing in browser

### ISSUE THREAD

**Issue by untitaker** [pockethub/PocketHub#200]

**Comment by larsgrefer** [comment]
I dont think that is what we want.
In fact we are catching links to github.com in order to show the resource in our app instead.

**Comment by untitaker** [comment]
Some things can't be handled by the app. Share option for the link would be better though.

**Comment by larsgrefer** [comment]
1. We are trying to handle as much as possible
2. We have a share action (at least for issues and repositories).

If you can find anything we can't handle, please open a more specific issue for that particular case.

If you can find any type of object with no share action available, please open an issue too.

**Comment by larsgrefer** [comment]
@hzsweers Whats your opinion?

**Comment by hzsweers** [comment]
I agree with the original issue, and we should add support for opening
things in the browser as a fall back option. There will always be things
that users will prefer to have a browser fallback option (new features,
broken things, etc).

Furthermore, Lars please don't speak for us about decisions like this. It's
fair to have a discussion, but the continued use of "we" gives the false
impression that you are a collaborator in the organization, and we don't
want to create confusion.

**Comment by larsgrefer** [comment]
Sorry for creating the confusion, with *"we"* I just meant *"all the people who are working on this app"* regardless of whether they have write access to this repository or not. I'll try to prevent this in future.

Back to topic: at the moment the activity which handles the URLs shows an error dialog if it's not able to parse the given URL. I think this behavior should be changed to just show a warning toast and to create a new chooser for the intent. If you like, I'll provide a pull request for that.
